tools/debian/control

```diff
@@ -28,24 +30,26 @@ Build-Depends: debhelper (>= 9),
                glib-networking,
                nodejs-legacy,
                npm,
-               openssh-client
-Standards-Version: 3.9.6
+               openssh-client <!nocheck>,
+Standards-Version: 3.9.8
 Homepage: http://cockpit-project.org/
 #Vcs-Git: git://anonscm.debian.org/collab-maint/cockpit.git
 #Vcs-Browser: https://anonscm.debian.org/cgit/collab-main/cockpit.git
 
 Package: cockpit
-Architecture: any
+Architecture: all
 Depends: ${misc:Depends},
-         cockpit-bridge (= ${binary:Version}),
-         cockpit-ws (= ${binary:Version}),
+         cockpit-bridge (>= ${source:Version}),
+         cockpit-bridge (<< ${source:Version}.1~),
+         cockpit-ws (>= ${source:Version}),
+         cockpit-ws (<< ${source:Version}.1~),
```

Comment:
That's precisely what this does, what do you mean? This is the usual way of specifying dependencies from an `Architecture: all` to an `Architecture: any` package, as the latter can get "binary NMUs", i. e. get rebuilt with a `b1` suffix on library transitions; but that  doesn't rebuild the `Arch: all` packages, so `${binary:Depends}` doesn't work in this case.